### PR TITLE
Standardizing Module specific READMEs

### PR DESCRIPTION
**Pull request by guptamukul-google**

### CODE REVIEW

File: docs/CONTRIBUTING.md [hunk 1]
```diff
@@ -38,23 +38,32 @@ The project has the following file structure:
...
+        - `# RAD LAB [MODULE-NAME] Module` : Title of the Module Specific README.
 
-```
-<!-- BEGIN TFDOC -->
-<!-- END TFDOC -->
-```
+    - SUB-HEADINGS:
+        - `## Module Overview` : Description about the module. 
+        - `## GCP Products/Services`: List the GCP products and the sevices used in the respective module development.
+        - `## Reference Architecture Diagram` : Add the GCP architecture diagram under this section. Images for module specific architechture diagrams are stored under [docs/images](./images/) directory.
```
Comment (acolver) [inline docs/CONTRIBUTING.md:59]: We either need a folder and standard name (e.g. `/[MODULE_NAME]/images/architecture.png`) or standardize on the module name being the file name (e.g. `docs/images/[MODULE_NAME]_architecture.png`)

File: docs/CONTRIBUTING.md [hunk 2]
```diff
@@ -38,23 +38,32 @@ The project has the following file structure:
...
+
+    NOTE: Add below 2 tags in the RAD-Lab module specific README.md for populating structured Variables and Outputs automatically via [tfdoc.py](../tools/tfdoc.py).
```
Comment (acolver) [inline docs/CONTRIBUTING.md:61]: Note: Add the below two tags ...

File: modules/NEON/README.md [hunk 3]
```diff
@@ -1,21 +1,23 @@
...
-* BigQuery
-* Analytics Hub
-* Cloud Storage
-* Billing Budget
+1. Vertex AI Workbench Notebooks
+2. Virtual Private Cloud (VPC)
+3. BigQuery
+4. Analytics Hub
+5. Cloud Storage
+6. Billing Budget
```
Comment (acolver) [inline modules/NEON/README.md:14]: I think the *unordered* (original) style makes more sense here.

File: docs/CONTRIBUTING.md [hunk 4]
```diff
@@ -38,23 +38,32 @@ The project has the following file structure:
...
 
-NOTE: Make sure all the variables are in alphabetical order in `variables.tf`
+    NOTE: Make sure all the variables are in alphabetical order in `variables.tf`
 
-- If any additional scripts (apart from **terraform** configs) are required to support the module either to *build* or *use* the module, move them to individual module's subdirectory, example: `/modules/MODULE-NAME/scripts/build` & `/modules/MODULE-NAME/scripts/usage` respectively. 
+- If any additional scripts (apart from **terraform** configs) are required to support the module either to *build* or *use* the module, move them to individual module's subdirectory, example: `/modules/[MODULE-NAME]/scripts/build` & `/modules/[MODULE-NAME]/scripts/usage` respectively. 
 
-- A README.md containing more information about the modules, required IAM permissions and any specific instructions to create the infrastructure.
+- A **README.md** containing more information about the modules, required IAM permissions and any specific instructions to create the infrastructure, etc. Make sure to add atleast below mentioned Headings & Sub Headings:
```
[the remark this answers is not in the document]
Comment (guptamukul-google) [reply docs/CONTRIBUTING.md:51]: Done

File: docs/CONTRIBUTING.md (hunk 2 above)
Comment (guptamukul-google) [reply docs/CONTRIBUTING.md:61]: Done

File: modules/NEON/README.md (hunk 3 above)
Comment (guptamukul-google) [reply modules/NEON/README.md]: Done

File: docs/CONTRIBUTING.md (hunk 1 above)
Comment (guptamukul-google) [reply docs/CONTRIBUTING.md]: Moved GCP architecture diagram under `modules/[MODULE_NAME]/images` with filename as `architecture.png`. Also updated the **CONTRIBUTING.md** for future contributions.

File: modules/NEON/README.md [hunk 5]
```diff
@@ -15,7 +17,7 @@ This RAD Lab module provides a managed environment for research and data analyti
 
 Below Architechture Diagram is the base representation of what will be created as a part of [RAD Lab Launcher](../../radlab-launcher/radlab.py).
 
-![](../../docs/images/V9_NEON.png)
+![](./images/architecture.png)
```
Comment (acolver) [inline modules/NEON/README.md:20]: This file is not present.

Comment (acolver) [reply modules/NEON/README.md:20]: Looks like it got put in a new folder `neon/images/` instead of `NEON/images/`

**Review by acolver — CHANGES_REQUESTED** [review]

File: modules/NEON/README.md (hunk 5 above)
Comment (guptamukul-google) [reply modules/NEON/README.md:20]: Resolved

**Pull request merged**